PROJECT

```diff
@@ -27,6 +27,9 @@ resources:
   kind: DRPolicy
   path: github.com/ramendr/ramen/api/v1alpha1
   version: v1alpha1
+  webhooks:
+    validation: true
+    webhookVersion: v1
```

Comment:
Can we stick to v1alpha1 as in the other CRDs? (applies to DRCluster webhook as well)